roles/tm-deepsec-aws-setup/tasks/main.yml

```diff
@@ -0,0 +1,86 @@
+---
+# tasks file for tm_silent_setup
+
+- name: Ensure bash, OpenSSl, and libssl are the latest versions
+  yum: name={{ item }} update_cache=true state=latest
+  with_items: "{{ pre_package }}"
+  tags: packages
+
+- name: Install psycopg2 python package for postgres
+  pip:
+    name: "{{ item }}"
+  with_items: "{{ pip_package }}"
+  #become_user: ec2-user
+
+- name: Change psycopg2 dir permission to 0755
+  file:
+    path: "{{ item }}"
+    mode: u=rwX,g=rX,o=rX
+    recurse: yes
+  with_items: "{{ dir_permission }}"
+
+- name: Remove the postgresql package
+  yum:
+    name: "{{ item }}"
+    state: present
+  with_items: "{{ uninstall_postgres }}"
+
+- name: Remove pgsql directory
+  file:
+    path: "{{ item }}"
+    state: absent
+  with_items: "{{ remove_dir }}"
+
+- name: Install postgres 11 libs rpm from a remote file
```

Comment:
I would prefer you set-up the yum repository with `yum_repository`. Also, can you use the RHEL packages instead?